test/unit/spec/EnrollWindowsHello_spec.js

```diff
@@ -228,6 +228,7 @@ function (Okta,
           test.form.submit();
           return Expect.waitForSpyCall(webauthn.makeCredential, test);
         })
+        .then(tick)
```

Comment:
same here, we should wait for something more specific. This test seems to be waiting for the spy call, if that is not enough, then let's try to wait for something else, like a controller

Reply:
Removed